netman/adapters/switches/backward_compatible_brocade.py

```diff
@@ -43,20 +43,17 @@ def set_access_mode(self, interface_id):
     def remove_trunk_vlan(self, interface_id, vlan):
         super(BackwardCompatibleBrocade, self).remove_trunk_vlan(_add_ethernet(interface_id), vlan)
 
-    def remove_native_vlan(self, interface_id):
-        return super(BackwardCompatibleBrocade, self).remove_native_vlan(_add_ethernet(interface_id))
+    def unset_native_vlan(self, interface_id):
```

Comment:
according to nomenclature:

Name should contain parent resource (e.g. interface_port_mode, interface_access_vlan)

therefore shouldn't this be unset_interface_native_vlan ?